docs/source/platforms/radxa/rock4.rst

```diff
@@ -0,0 +1,36 @@
+.. |yes| image:: ../../images/yes.png
+.. |no| image:: ../../images/no.png
+
+.. role:: underline
+   :class: underline
+
+rock4
+=====================
+
++----------------+-----------------+
+| System on Chip | Rockchip 3399   |
++----------------+-----------------+
+| Number GPIO    | 27              |
++----------------+-----------------+
+| Status         | support in part |
```

Comment:
Can you describe in the comments here what isn't supported because below, the supported features all say yes.

Reply:
Also, `partly supported` is proper English.